src/index.js

```diff
@@ -1,8 +1,9 @@
 import React, { Component } from 'react';
 import csjs from 'csjs';
+import scopedName from 'csjs/lib/scoped-name';
```

Comment:
I'm concerned about pulling in a "private" lib from csjs. I wouldn't be as uneasy about this if we were to ship CSJS along with this package but I can be convinced. Thoughts here?